id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/controller/IdRepoController.java

```diff
@@ -402,7 +410,6 @@ public ResponseEntity<IdResponseDTO> updateAuthtypeStatus(
 					: getIdType(individualId).getIdType());
 			IdType idType = validator.validateIdType(authTypeStatusRequest.getIndividualIdType());
 			validator.validateIdvId(individualId, idType);
-			validator.validateAuthTypes(authTypeStatusRequest.getRequest());
```

Comment:
I think this is not related to resident changes. Please keep it if it is there in release.

Reply:
Done